### DISCUSSION TITLE
[合作委托] 佩里科岛抢劫任务四缺三

### DISCUSSION THREAD

**Discussion by TimeMki** [share121/inter-knot#223] [其他]
![Uploading 705d777678fae7921d2468a6d2ef3a7e.png…]()
抢劫国内最大的银行？去过了。勇闯赌场金库？做过了。潜入冷酷毒枭金发老大戒备森严的私人派对海岛？这才像话嘛。离开洛圣都的灯红酒绿，前往佩里科岛的沙滩。您可以【单独】冒险，或【最多与三名其他玩家】一同从头玩到尾，探索多种侵入方法，并抢劫一个全新的地点以捞得大笔财富。您的旅程始于洛圣都夜生活的最新场所，您将在那里认识一位全新联系人…
目标：一架私人喷气机将带着一群DJ飞往佩里科岛，而米盖尔在这架飞机上预定了一个座位，这正是您入侵和侦察这个金钱可以买到的最安全的据点的机会。抵达后，悄悄离开派对，并找到藏有文件的豪宅。避开武装保安和监控摄像头，并记下所有弱点，比如隐藏入口、出口或其他以后可能会派上用场的安全漏洞。保持低调，尽可能收集更多信息，以提升成功的机会。

**Comment by TimeMki** [comment]
![Uploading 705d777678fae7921d2468a6d2ef3a7e.png…]()

**Comment by kangkangMike** [comment]
啊哈，ccmt
